setup.py

```diff
@@ -9,7 +9,7 @@
 # allow setup.py to be run from any path
 os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
 
-REQUIREMENTS = ['django>=1.8', 'django-cas-ng', 'ldap3']
+REQUIREMENTS = ['django>=1.8,<2.1', 'django-cas-ng', 'ldap3']
```

Comment:
Does it break under 2.1 ? Should we create an issue or something to document that? We should probably also note tested versions in the readme.

Reply:
Yes, let's note that so we can easily check when django-cas-ng supports 2.1